### PR TITLE
- gcemetadata

### PR DESCRIPTION
**Pull request by rjschwei**
+ Add a man page

### CODE REVIEW

**Comment by rjschwei** [comment]
@jbyers-suse @joelgordon  would you please read the man page and see if this makes sense to you. Thanks,

File: gcemetadata/man/man1/gcemetadata.1 [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+.TH gcemetadata.1
+.SH NAME
+gcemetadata \- Query the Google Compute Engine metadata server
+.SH SYNOPSIS
+.B gcemetadata [option]
+.SH DESCRIPTION
+.B gcemetadata
+is used to query the Google Compute Engine metadata server. The available
+data is documented at 
+https://cloud.google.com/compute/docs/storing-retrieving-metadata. The
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: I recommend leaving a space between the URL and the period at the end of the sentence, as including the period in the URL results in a 404 NOT FOUND.

File: gcemetadata/man/man1/gcemetadata.1 [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+command line options the tool accepts are for the most part dynamically and
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: "for the most part dynamically and" ?
Something's missing here.

File: gcemetadata/man/man1/gcemetadata.1 [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+command line options other than
+.I --output
+or
+.I --xml
+are specified. If any specific option is given the
+.I --query
+option must be given. If a specific query does not produce any value or
+the query results in an error the data for the requested option will be
+set to
+.I unavailable.
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: Consider `.IR unavailable .`

File: gcemetadata/man/man1/gcemetadata.1 [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+.SH OPTIONS
+.IP "-a --api"
+Specify the version of the metadata server API to use.
+.IP "--help"
+Print all available command line options to standard out.
+.IP "--identity-format"
+Set the format for the identity token. Supported values are
+.I full
+or
+.I standard.
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: Consider `.IR standard .` as the period is not part of the supported value.

File: gcemetadata/man/man1/gcemetadata.1 [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+, providing the smallest possible amount of information.
+.IP "--listapis"
+List the available api versions.
+.IP "--output"
+Write the requested information to the given file.
+.IP "--query"
+Specify whether to query the project or the instance, possible values are
+.I project
+or
+.I instance.
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: Consider `.IR instance .`

File: gcemetadata/man/man1/gcemetadata.1 [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+If any option, other than
+.I --output
+or
+.I --xml
+is specified the
+.I --query
+option is required.
+.IP "--version"
+Print he version of the tool.
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: s/he/the/

File: gcemetadata/man/man1/gcemetadata.1 [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+Will print the external IPv4 address of the instance.
+.SH CAVEAT
+When requesting all possible information, i.e no argument or only
+.I --output
+and/or
+.I --xml
+are given the identity information in the resulting output will be shown
+as
+.I unavailable.
```
Comment (bear454) [inline gcemetadata/man/man1/gcemetadata.1]: `.IR unavailable .`

**Review by bear454 — CHANGES_REQUESTED** [review]

**Pull request merged**